match-standards-mix01.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Identifying standards from mix01"
+author: "Andrea Vicini, Johannes Rainer"
+output:
+  rmarkdown::html_document:
+    highlight: pygments
+    toc: true
+    toc_depth: 3
+    fig_width: 5
+---
+
+# Introduction
+
+Mixes of standards have been solved in water or added to human serum sample
+pools in two different concentration and these samples were measured with the
+LC-MS setup from Eurac (used also to generate the CHRIS untargeted metabolomics
+data). The goal of this analysis is now to:
+
+- Determine the retention time of each standard in water and serum.
+- Define which ions/adducts are measured for each standard (with relative
+  abundances, i.e. which is most highly abundant ion etc).
+- Extract all MS/MS spectra for each standard and store that in a database.
+
+# Identifying standards in sample mix 01
+
+- load standards
+- calculate exact mass for each standard (based on the provided formula)
+- subset to mix01
+
+## Data import
+
+- load the data (separate for pos and neg, water and serum)
+
+## Chromatographic peak detection
+
+- perform chromatogramphic peak detection.
+
+## Correspondence analysis
+
+- perform correspondence analysis.
+
+## Identification of features matching standards
+
+- identify all features matching one adduct of a standard (use `matchMz`)
+- subsection for each standard
+
+
+### Standard:
```

Comment:
i.e. you will need to create one subsection for each individual standard.

Reply:
What this "subsection for each standard" will contain? Just the table of the matches restricted to that given standard?

Reply:
exactly. So, one subsection with the results for one standard: which features would match, find MS2 spectra for these features, match them against reference databases.

Reply:
Then I'm not sure I understand completely why we need these subsections. After matching features to adducts of standards (we could do it with a single `matchMz` call right?) we would have a table that we can sort according to the standards name. Then we extract the MS2 data for each feature in the table and we fill another column of the table with the matches found in mass bank for each feature. Wouldn't results be visible also from that?

Reply:
OK, I see. Do as you think it would be best. Can also be one general table with the results from a single `matchMz` call. Only, I think it would be easier to check if then the MS2 matching would be performed seprately for each standard. Because maybe we might then also want to have a section for a standard for which nothing was found. And then I would like to manually dig into the data for that to see why we don't find anything...